emote/mixins/logging.py

```diff
@@ -47,15 +47,14 @@ def log_windowed_scalar(
         whichever length is found first will be permanent.
         """
 
-        # we allow windowed[100]:some_key/foobar to override window size
-        if "windowed[" in key:
-            p, k = key.split(":")
-            length = int(p.split("[")[1][:-1])
-            key = k
-        else:
-            length = self._default_window_length
-
         if key not in self.windowed_scalar:
+            # we allow windowed[100]:some_key/foobar to override window size
+            if "windowed[" in key:
+                p, k = key.split(":")
```

Comment:
k isn't actually used anymore, is it?

Reply:
You are right, I forgot to update that line. Changed it to `p = key.split(":")[0]` :)